etc/spack/defaults/windows/bootstrap-resource.yaml

```diff
@@ -0,0 +1,16 @@
+bootstrap-resource:
+  # If set to false Spack will not fetch bootstrap resources but instead
+  # proceed without fetching said resources
+  enable: true
+  # Trusted sources to acquire bootstrap resources from
+  resources:
+    gpg:
+      providers:
+        - name: Github Actions
+          endpoint: https://spack.github.io/windows-bootstrap-resources/resources/gpg4win/gpg4win.tar.gz
+          sha256: notasha
```

Comment:
Yes, I'm just in the process of updating the workflow that produces them so I don't have actual sha's right now. I thought this would be obvious enough to make sure we won't forget before a merge.

Reply:
I have a sha for gpg, just not file, not sure what changed there but I'm suddenly missing a dep. Should be resolved before we're ready to hit merge here.